web/package.json

```diff
@@ -30,6 +32,7 @@
     "chart.js": "2.7.2",
     "core-js": "2.5.3",
     "file-saver": "1.3.3",
+    "hammerjs": "^2.0.8",
```

Comment:
please remove hammerjs from package json since it's only for controlling touch behaviors